src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -179,18 +180,31 @@ else if( this.Name.equals( "PatternTerminal.Substitute" ) )
 				cpt.getPatternTerminal().setSubstitution( this.Value.equals( "1" ) );
 			}
 		}
-		else if( this.Name.startsWith( "StorageBus." ) && c instanceof ContainerStorageBus )
+		else if( this.Name.startsWith( "StorageBus." ) )
 		{
-			final ContainerStorageBus ccw = (ContainerStorageBus) c;
 			if( this.Name.equals( "StorageBus.Action" ) )
 			{
 				if( this.Value.equals( "Partition" ) )
 				{
-					ccw.partition();
+					if( c instanceof ContainerStorageBus )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS134)